dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -176,10 +176,13 @@ public static GitHubBuilder getBuilder(String cacheNamespace) {
         OkHttpClient build = builder.build();
         // Must set the cache max age otherwise kohsuke assumes 0 which significantly slows down our GitHub requests
         OkHttpGitHubConnector okHttp3Connector = new OkHttpGitHubConnector(build, GITHUB_MAX_CACHE_AGE_SECONDS);
-        return new GitHubBuilder()
-            .withRateLimitChecker(new LiteralValue(SLEEP_AT_RATE_LIMIT_OR_BELOW))
+        GitHubBuilder gitHubBuilder = new GitHubBuilder()
```

Comment:
The master branch has a `withRateLimitHandler(new FailRateLimitHandler(githubTokenUsername))` -- does that need to be restored for the non-CircleCI case?

Reply:
Good catch, added back handler.